R/scale-.r

```diff
@@ -888,68 +890,16 @@ ScaleDiscrete <- ggproto("ScaleDiscrete", Scale,
-#'   - `waiver()` for the default breaks computed by the
-#'     transformation object
-#'   - A numeric vector of positions
-#'   - A function that takes the limits as input and returns breaks
-#'     as output
-#' @param minor_breaks One of:
-#'   - `NULL` for no minor breaks
-#'   - `waiver()` for the default breaks (one minor break between
-#'     each major break)
-#'   - A numeric vector of positions
-#'   - A function that given the limits returns a vector of minor breaks.
-#' @param n_breaks An integer guiding the number of major breaks. The algorithm
-#'   may choose a slightly different number to ensure nice break labels. Will
-#'   only have an effect if `breaks = waiver()`. Use `NULL` to use the default
-#'   number of breaks given by the transformation.
-#' @param labels One of:
-#'   - `NULL` for no labels
-#'   - `waiver()` for the default labels computed by the
-#'     transformation object
-#'   - A character vector giving labels (must be same length as `breaks`)
-#'   - A function that takes the breaks as input and returns labels
-#'     as output
-#' @param limits A numeric vector of length two providing limits of the scale.
-#'   Use `NA` to refer to the existing minimum or maximum.
-#' @param rescaler  Used by diverging and n colour gradients
-#'   (i.e. [scale_colour_gradient2()], [scale_colour_gradientn()]).
-#'   A function used to scale the input values to the range \[0, 1].
-#' @param oob Function that handles limits outside of the scale limits
-#'   (out of bounds). The default replaces out of bounds values with `NA`.
-#' @inheritParams scale_x_discrete
-#' @param na.value Missing values will be replaced with this value.
-#' @param trans Either the name of a transformation object, or the
-#'   object itself. Built-in transformations include "asn", "atanh",
-#'   "boxcox", "exp", "identity", "log", "log10", "log1p", "log2",
-#'   "logit", "probability", "probit", "reciprocal", "reverse" and "sqrt".
-#'
-#'   A transformation object bundles together a transform, its inverse,
-#'   and methods for generating breaks and labels. Transformation objects
-#'   are defined in the scales package, and are called `name_trans`, e.g.
-#'   [scales::boxcox_trans()]. You can create your own
-#'   transformation with [scales::trans_new()].
-#' @param guide A function used to create a guide or its name. See
-#'   [guides()] for more info.
-#' @param position The position of the axis. "left" or "right" for vertical
-#' scales, "top" or "bottom" for horizontal scales
-#' @param super The super class to use for the constructed scale
-#' @keywords internal
-continuous_scale <- function(aesthetics, scale_name, palette, name = waiver(),
-  breaks = waiver(), minor_breaks = waiver(), n_breaks = NULL, labels = waiver(),
-  limits = NULL, rescaler = rescale, oob = censor, expand = waiver(), na.value = NA_real_,
-  trans = "identity", guide = "legend", position = "left", super = ScaleContinuous) {
+ScaleBinned <- ggproto("ScaleBinned", Scale,
```

Comment:
What's up with the big diff here?

Reply:
You must be looking at the code from before I merged master in? `scale-.r` was reorganised while this PR was developed